PrimeNumberNaive/FunctionNaive.java

```diff
@@ -0,0 +1,40 @@
+/*
+program realized naive test of Prime Number
+*/
+
+public class FunctionNaive {
+	
+	private double startValue = 2;
+	private double result=0;
+	private double number;
+	private double copyResult;
+	private double floorDoubleResult;
+	private int floorResult;
+
+	public void setFunction(double num) {
+		number = num;
+	}
+
+	public void getFunction() {
+	while(result <= (number - 1)) {
```

Comment:
why '-1'?

Reply:
It's result form Prime Number properties. Prime Number divide by 1 and divide by itself (n), that;s why we divide in range 2..n-1